src/user/models/user/user.service.spec.ts

```diff
@@ -146,4 +146,22 @@ describe('UserService', () => {
 
     await expect(service.getUserBuyFee(1, 23467)).resolves.toStrictEqual({ fee: 0.5, refBonus: 0 });
   });
+
+  it('should return a fee of 1.2 and refBonus of 0 for crypto routes, if no ref was used', async () => {
+    setup(AccountType.PERSONAL, undefined, undefined, '000-000');
+
+    await expect(service.getUserCryptoFee(1)).resolves.toStrictEqual({ fee: 1.2, refBonus: 0 });
+  });
+
+  it('should return a fee of 1.2 and refBonus of 0 for crypto routes, if ref is undefined', async () => {
+    setup(AccountType.PERSONAL);
+
+    await expect(service.getUserCryptoFee(1)).resolves.toStrictEqual({ fee: 1.2, refBonus: 0 });
+  });
+
+  it('should return a fee of 1.1 and refBonus of 0.1 for crypto routes, if ref was used', async () => {
+    setup(AccountType.PERSONAL, undefined, undefined, '000-001');
+
+    await expect(service.getUserCryptoFee(1)).resolves.toStrictEqual({ fee: 1.1, refBonus: 0.1 });
+  });
```

Comment:
Add a unit test for individual crypto fee?

Reply:
fair enough. will add one